content/docs/concepts/security/index.md

```diff
@@ -771,6 +770,51 @@ spec:
     name: "products-viewer"
 {{< /text >}}
 
+### Using Istio authorization on plain TCP protocols
+
+The above examples showed a typical way to use Istio authorization on services using HTTP protocol.
+In that case, all fields in a `ServiceRole` and `ServiceRoleBinding` are supported.
+
+Istio authorization also supports services using any plain TCP protocols, such as mongodb. In this case,
+you specify the `ServiceRole` and `ServiceRoleBinding` the same way as for a HTTP service, except that
+you cannot use the fields that are only applicable to HTTP services, which includes `paths` and `methods`
+in `ServiceRole`, and `group` in `ServiceRoleBinding`.
+
+For the supported constraints and properties that could be used for TCP service, please refer to the
```

Comment:
Passive voice and consistency issues. You refer to them as fields before and now you call them constraints and properties, this is confusing for readers. Stick to the terminology and use is across the entire document. I suggest:
To review the fields supported for TCP services, refer to the [constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).

Reply:
The constraints and properties are a different from the fields, I listed it separately.